src/libraries/System.Formats.Cbor/tests/PropertyTests/CborPropertyTests.cs

```diff
@@ -178,6 +178,7 @@ static int GetLengthEncodingLength(int length)
         }
 
         [Property(Replay = ReplaySeed, MaxTest = MaxTests, Arbitrary = new[] { typeof(CborRandomGenerators) })]
+        [ActiveIssue("", TestPlatforms.iOS | TestPlatforms.tvOS)]
```

Comment:
What problem was hit with this test? Is it an unrelated test failure? We should attach an issue with this label to not forget about it or address in a separate PR